### ISSUE TITLE
Accessibility: MAS20: alt text  is empty for the videos.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#3]
#A11yMAS
#MAS20
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs/introvideos/overview
2.Press F12 key and check the alt text for the videos under "Introductory Videos" below the header of the page.

**Actual Result:**
The alt text is empty for the videos under "Introductory Videos" below the header of the page.

**Expected Result:**
There should be alt text for the videos under "Introductory Videos" below the header of the page.

**User Impact:**
Users with memory and attention limitations benefit by being able to quickly rediscover their context after an interruption.

![alt text is empty for the videos](https://cloud.githubusercontent.com/assets/24825441/21672241/26391254-d347-11e6-8f29-03ddb6dc7996.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/a3677c1fcb4624b76a3a42e0bd2a09ea67b7365b
